### PR TITLE
Ft create user profile page

### PR DESCRIPTION
**Pull request by KapsonLabs**
User profile page where the user can view a summary of his intervention/ red flag records.

### CODE REVIEW

**Comment by llwasampijja** [comment]
So far so good, except many links are not yet working. Otherwise Nice work!

**Comment by llwasampijja** [comment]
Additionally, I have noticed you haven't added responsiveness yet! Don't forget that!

File: UI/css/main.css [hunk 1]
```diff
@@ -237,3 +237,60 @@ footer{
     /* position: relative; */
     display: contents;
 }
+
+/*PROFILE PAGE*/
+
+#profile .redflag-detail{
+    float: right;
+    width: 60%;
+    /* border: 1px solid black; */
```
Comment (kyakusahmed) [inline UI/css/main.css:246]: commented code bro

**Comment by kyakusahmed** [comment]
otherwise , i love ur html

**Comment by KapsonLabs** [comment]
Thanks @llwasampijja. I usually add the responsiveness at the end of the entire HTML. Working around the clock to get most of my links working.

**Comment by masete** [comment]
hey @KapsonLabs can u pliz maybe send me your gh-pages link, its hard to review the ui with out looking at how its looking thanks

**Comment by kalsmic** [comment]
@KapsonLabs nice colour balancing. Looking through your pages I notice that you have a clear understanding of the project. I had so little hardship navigating around the site i.e the home link is not consistent on all the pages.

**Pull request merged**